diff --git a/LoopCore/LoopCoreConstants.swift b/LoopCore/LoopCoreConstants.swift
index d56f2ab9b6..9158bb5938 100644
--- a/LoopCore/LoopCoreConstants.swift
+++ b/LoopCore/LoopCoreConstants.swift
@@ -16,7 +16,7 @@ public enum LoopCoreConstants {
     /// The amount of time in the future a glucose value should be considered valid
     public static let futureGlucoseDataInterval = TimeInterval(minutes: 5)
 
-    public static let defaultCarbAbsorptionTimes: CarbStore.DefaultAbsorptionTimes = (fast: .minutes(30), medium: .hours(3), slow: .hours(5))
+    public static let defaultCarbAbsorptionTimes: CarbStore.DefaultAbsorptionTimes = (fast: .minutes(90), medium: .hours(2), slow: .hours(3))
 
     /// How much historical glucose to include in a dosing decision
     /// Somewhat arbitrary, but typical maximum visible in bolus glucose preview
